Lib/test/test_exceptions.py

```diff
@@ -674,6 +674,14 @@ def test_str(self):
     def testExceptionCleanupNames(self):
         # Make sure the local variable bound to the exception instance by
         # an "except" statement is only visible inside the except block.
+        try:
+            raise Exception()
+        except Exception as e:
+            self.assertTrue(e)
```

Comment:
Could change both to self.assertIsNotNone(e)

Reply:
Yeah, I just copy-pasted the existing code. I don't think it really matters what it does there, just `e` would be just as fine, because I am sure there are other tests which test that the caught exception is indeed an exception :)